scmrepo/git/backend/dulwich/__init__.py

```diff
@@ -491,23 +499,20 @@ def get_refs_containing(self, rev: str, pattern: Optional[str] = None):
     def push_refspec(
         self,
         url: str,
-        src: Optional[str],
-        dest: str,
+        refspecs: Union[str, Iterable[str]],
```

Comment:
the function signature needs to be updated in `backend/base.py` and the other backends